libcudacxx/include/cuda/std/__atomic/types/base.h

```diff
@@ -64,11 +64,17 @@ struct __atomic_storage
   }
 };
 
+#if defined(_CCCL_CUDA_COMPILER)
+extern "C" _CCCL_DEVICE void __atomic_is_not_supported_pre_sm_60();
```

Comment:
Suggestion: Consider adding a comment that there is no definition on purpose, causing a linker error.

Reply:
That is a very common pattern we use at a lot of places, I dont think we should add comments everywhere when this happens

Reply:
Alright.